repos/fdbt-types/matchingJsonTypes.ts

```diff
@@ -251,6 +253,7 @@ export interface ReturnTicket extends BasePointToPointTicket {
     outboundFareZones: FareZone[];
     returnPeriodValidity?: ReturnPeriodValidity;
     additionalServices?: AdditionalService[];
+    cap?: CapSelection;
```

Comment:
I think we just need to add cap?: CapSelection to BaseTicket, as all other tickets will inherit it.